dora/core/client/fs/src/test/java/alluxio/client/file/ufs/UfsBaseFileSystemTest.java

```diff
@@ -356,18 +345,4 @@ public void writeThenGetStatus() throws IOException, AlluxioException {
     }
     Assert.assertEquals(chunkSize, mFileSystem.getStatus(uri).getLength());
   }
-
-/*  @Test
-  public void getStatusWhenWriting() throws IOException, AlluxioException {
```

Comment:
Now the FileOutStream does not flush data to ufs until the stream close, so I delete this commented method.

Reply:
Could you please take a look ? @LuQQiu Thanks!

Reply:
from alluxio fuse level we need to support - write 10 bytes - get status 10 bytes - write 10 bytes - get status 20 bytes workflow to keep the end application succeed.
but it's not required on UfsBaseFileSystem level since different ufs has different write patterns and may not support this workflow

Reply:
OK, thanks! I will remove this method later.